tests/integration/test_filter.py

```diff
@@ -96,7 +91,6 @@ def test_filter_cast_date(c, input_table, request):
     assert_eq(return_df, expected_df)
 
 
-@pytest.mark.skip(reason="WIP DataFusion")
 @pytest.mark.parametrize(
     "input_table",
     [
```

Comment:
gpuCI is failing on this test and would probably need to same changes as above.

Reply:
ahh ok will make those as skip for now and figure out what is going on. Thanks for the review

Reply:
Oh cool. Didn't know I could do that. Just made the change and thanks for the suggestion.